yt/yt/ytlib/api/native/client_queries_impl.cpp

```diff
@@ -1074,4 +1081,58 @@ void TClient::DoAlterQuery(TQueryId queryId, const TAlterQueryOptions& options)
 
 ////////////////////////////////////////////////////////////////////////////////
 
+TGetQueryTrackerInfoResult TClient::DoGetQueryTrackerInfo(const TGetQueryTrackerInfoOptions& options)
+{
+    IClientPtr client;
+    TString root;
+    std::tie(client, root) = GetNativeConnection()->GetQueryTrackerStage(options.QueryTrackerStage);
+
+    YT_LOG_DEBUG(
+        "Getting query tracker information (Attributes: %v)",
+        options.Attributes);
+
+    auto attributes = options.Attributes;
+
+    attributes.ValidateKeysOnly();
+
+    TString clusterName = "";
+
+    if (attributes.AdmitsKeySlow("cluster_name")) {
+        YT_LOG_DEBUG("Getting cluster name");
+        clusterName = client->GetClusterName().value_or("");
+    }
+
+    TNodeExistsOptions nodeExistsOptions;
+    nodeExistsOptions.ReadFrom = EMasterChannelKind::Cache;
+    bool accessControlSupported = WaitFor(client->NodeExists(TString(QueriesAcoNamespacePath), nodeExistsOptions))
+        .ValueOrThrow();
+
+    static const TYsonString EmptyMap = TYsonString(TString("{}"));
+    TYsonString supportedFeatures = EmptyMap;
+    if (attributes.AdmitsKeySlow("supported_features")) {
+        supportedFeatures = BuildYsonStringFluently()
+            .BeginMap()
+                .Item("access_control").Value(accessControlSupported)
```

Comment:
I'll clarify just in case. Do I understand correctly that if we create the qt aco namespace, we are claiming that both qt and proxy are fairly new?